modules/ROOT/pages/syntax-quick-reference.adoc

```diff
@@ -0,0 +1,1292 @@
+
+== Formatted text
+
+.Bold, italic, and monospace
+[#ex-format]
+----
+include::text:example$text.adoc[tag=b-bold-italic-mono]
+----
+
+.View result of <<ex-format>>
+[%collapsible.result]
+====
+include::text:example$text.adoc[tag=b-bold-italic-mono]
+====
+
+.Highlight, underline, and strikethrough
+[#ex-lines]
+----
+include::text:example$text.adoc[tag=qr-all]
+----
+
+.View result of <<ex-lines>>
+[%collapsible.result]
+====
+include::text:example$text.adoc[tag=qr-all]
+====
+
+.Superscript and subscript
+[#ex-sub-sup]
+----
+include::text:example$text.adoc[tag=b-sub-sup]
+----
+
+.View result of <<ex-sub-sup>>
+[%collapsible.result]
+====
+include::text:example$text.adoc[tag=b-sub-sup]
+====
+
+.Smart quotes and apostrophes
+[#ex-curved]
+----
+include::text:example$text.adoc[tag=b-c-quote]
+----
+
+.View result of <<ex-curved>>
+[%collapsible.result]
+====
+include::text:example$text.adoc[tag=b-c-quote]
+====
+
+== Document header
+
+A xref:document:header.adoc[document header] is optional.
+The header may not contain blank lines and must be separated from the content by at least one blank line.
+
+.Title
+----
+include::document:example$title.adoc[tag=qr-title]
+----
+
+.Title and author line
+----
+include::document:example$header.adoc[tag=qr-author]
+----
+
+.Title, author line, and revision line
+----
+include::document:example$header.adoc[tag=qr-rev]
+----
+
+IMPORTANT: You cannot have a xref:document:revision-line.adoc[revision line] without an xref:document:author-line.adoc[author line].
+
+.Document header with attributes
+----
+include::document:example$header.adoc[tag=qr-attributes]
+----
+
+[#section-titles]
+== Section titles
+
+When the document type is `article` (the default), the document can only have one level-0 section title (`=`).
+This level-0 section title (i.e., the document title) must be in the document header.
+
+.Article doctype section levels
+[#ex-article]
+----
+include::sections:example$section.adoc[tag=base]
+----
+
+.View result of <<ex-article>>
+[%collapsible.result]
+====
+include::sections:example$section.adoc[tag=b-base]
+====
+
+The `book` document type can have additional level-0 section titles, which are interpreted as xref:sections:parts.adoc[parts].
+The presense of at least one part implicitly makes the document a multi-part book.
+
+.Book doctype section levels
+----
+include::sections:example$section.adoc[tag=book]
+----
+
+xref:sections:title-links.adoc#anchor[sectanchors]::
+When the document attribute `sectanchors` is set, a floating section icon anchor appears in front of the section title on hover.
+This feature enables a reader to easily bookmark the section.
+Section title anchors depend on support from the stylesheet to render properly.
+
+xref:sections:title-links.adoc#link[sectlinks]::
+When the document attribute `sectlinks` is set, the section titles become self-links.
+This feature enables a reader to easily bookmark the section.
```

Comment:
Since we are using the same sentence twice, it's not clear if I need to add both `sectanchors` and `sectlinks` to enable this feature.

Maybe we should describe `sectlinks` first which does one thing and then describe `sectanchors` which does what sectlinks does + adds a floating section icon anchor. What do you think?

Reply:
sectanchors doesn't build on sectlinks. They are discrete features and both provide the same benefit mentioned. But I question whether this information should even be in the syntax quick reference. I think we should just drop it.

Reply:
I revised the text, then commented out until we determine whether it belongs in the first place.